cypress/integration/collections_spec.js

```diff
@@ -38,7 +38,7 @@ describe('Dashboard Collections Page', () => {
 
     it('should display a link to view collections', () => {
       cy.contains('nav li a', 'Collections').as('collections');
-      cy.get('@collections').should('have.attr', 'href', '/collections');
+      cy.get('@collections').should('have.attr', 'href').and('match', /\/collections/);
```

Comment:
Is it possible to add `?startDateTime` to this match, just so we know to expect it?  NBD if not.

Reply:
Maybe only update one, because they're on all the pages.

Reply:
With the new one of doing this, I was able to revert back most tests that had to be changed, but updated this for Collections spec